Kiosk/Bid Fulfillment/BidCheckingNetworkModel.swift

```diff
@@ -33,48 +54,35 @@ class BidCheckingNetworkModel: NSObject {
         // If polling fails, we can still show bid confirmation. Do not error.
         } .catchTo( RACSignal.empty() )
     }
-
-    private func pollForUpdatedSaleArtwork() -> RACSignal {
-
-        let beginningBidCents = fulfillmentController.bidDetails.saleArtwork?.saleHighestBid?.amountCents ?? 0
-
-        let updatedSaleArtworkSignal = getUpdatedSaleArtwork().flattenMap { [weak self] (saleObject) -> RACStream! in
+    
+    private func pollForUpdatedBidderPosition(bidderPositionId: String) -> RACSignal {
+        let updatedBidderPosition = getUpdatedBidderPosition(bidderPositionId).flattenMap { [weak self] bidderPositionObject in
             self?.pollRequests++
-
+            
             logger.log("Polling \(self?.pollRequests) of \(self?.maxPollRequests) for updated sale artwork")
-
-            let saleArtwork = saleObject as? SaleArtwork
-            let updatedBidCents = saleArtwork?.saleHighestBid?.amountCents ?? 0
-
+            
             // TODO: handle the case where the user was already the highest bidder
```

Comment:
@ashfurrow @acjay There are tasks run as cron jobs, but they can be triggered manually as well. For the auction it’s: http://joe.artsy.net:9000/job/gravity-create-auction-on-staging-cron/